src/lake/Lake/Build/Library.lean

```diff
@@ -73,6 +73,19 @@ def LeanLib.staticFacetConfig : LibraryFacetConfig staticFacet :=
 def LeanLib.staticExportFacetConfig : LibraryFacetConfig staticExportFacet :=
   mkFacetJobConfig (LeanLib.recBuildStatic · true)
 
+@[specialize] protected def LeanLib.buildStaticFat
+(self : LeanLib) : FetchM (BuildJob FilePath) := do
+    withRegisterJob s!"{self.name}:static.fat" do
+    let mods ← self.rootModules.concatMapM fun mod => do
+      return (← mod.transImports.fetch).push mod
+    let oJobs ← mods.concatMapM fun mod =>
+      mod.nativeFacets (shouldExport := true) |>.mapM fun facet => fetch <| mod.facet facet.name
+    let libFile := self.staticFatLibFile
+    buildStaticLib libFile oJobs
```

Comment:
```suggestion
    buildStaticLib self.staticFatLibFile oJobs
```

Sorry for not catching this on the first review, but it seems stylistically best to do this all one one line.